### PR TITLE
fix kube client logs format

### PR DESCRIPTION
**Pull request by gy95**
1. fix some logs: should use `namespace/name` format but not use `name/namespace` format

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

Pull Request Attributes

Please check any characteristics that apply to this pull request. 

[ ] Does not have any [user-facing](https://github.com/istio/istio/tree/master/releasenotes#when-to-add-release-notes) changes. This may include CLI changes, API changes, behavior changes, performance improvements, etc.

### CODE REVIEW

File: pkg/kube/portforwarder.go [hunk 1]
```diff
@@ -39,7 +39,7 @@ type PortForwarder interface {
 	// Close this forwarder and release an resources.
 	Close()
 
-	// Block until connection closed (e.g. control-C interrupt)
+	// WaitForStop block until connection closed (e.g. control-C interrupt)
```
Comment (hzxuzhonghu) [inline pkg/kube/portforwarder.go:42]: blocks

File: pkg/kube/portforwarder.go [hunk 2]
```diff
@@ -30,7 +30,7 @@ import (
 
 // PortForwarder manages the forwarding of a single port.
 type PortForwarder interface {
-	// Run this forwarder.
+	// Start run this forwarder.
```
Comment (hzxuzhonghu) [inline pkg/kube/portforwarder.go:33]: runs

**Review by hzxuzhonghu — COMMENTED** [review]
Is there any difference?

File: pkg/kube/portforwarder.go (hunk 2 above)
Comment (gy95) [reply pkg/kube/portforwarder.go:33]: ok

**Comment by gy95** [comment]
> Is there any difference?

namespace should precede name, separeated by '/', more conventional

**Review by hzxuzhonghu — APPROVED** [review]
/ok-to-test

**Comment by hzxuzhonghu** [comment]
@gy95 You can apply for istio member, so that your pr can be tested without other members to label `/ok-to-test`

**Pull request merged**